synth.py

```diff
@@ -57,4 +59,12 @@
 # Format quoted strings as plain text.
 s.replace("google/cloud/bigquery_v2/proto/*.py", "[“”]", '``')
 
+# ----------------------------------------------------------------------------
+# Add templated files
+# ----------------------------------------------------------------------------
+templated_files = common.py_library(cov_level=100)
+# we do not want override the custom noxfile with the generated one
+os.remove(os.path.join(templated_files, "noxfile.py"))
```

Comment:
Looking at [CommonTemplates.py_library()](https://github.com/googleapis/synthtool/blob/master/synthtool/gcp/common.py#L56-L71), I suppose there is no better way of excluding a particular template file? Cannot manipulate the `excludes` list through `kwargs`...